internal/config/application_test.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReadConfig(t *testing.T) {
+	expUrlRequestQueue := "url_request_queue"
+	err := os.Setenv("QUEUE_URL_REQUEST_QUEUE", expUrlRequestQueue)
+	assert.Nil(t, err)
+
+	expScrResQueue := "storage-update-asd"
+	err = os.Setenv("QUEUE_SCRAPING_RESULT_QUEUE", expScrResQueue)
+	assert.Nil(t, err)
```

Comment:
Currently, the test passes because of using the default config value in the expected variable 'expScrResQueue'. If you set it to another value test will fail because of the non-matching name.
From the config, it has to be 'QUEUE_SAVE_ASD_REQUEST_QUEUE'.

So to check that the environment variable applies to configuration correctly, you need to change the value of the 'expScrResQueue' to any different from the default (for example, expScrResQueue := "test save queue") and change the environment variable name
from "QUEUE_SCRAPING_RESULT_QUEUE"
to ''QUEUE_SAVE_ASD_REQUEST_QUEUE"

Also it's better to change variable name too because it mentions old property name ( for example, expScrResQueue -> expSaveASDReqQueue)